### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by jaxmmmm**
Wow good features

### CODE REVIEW

File: lab-exercise-1/SarutaFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  public class SarutaFeature {
```
Comment (jaxmmmm) [inline lab-exercise-1/SarutaFeature.java:1]: good job.

Comment (jaxmmmm) [reply lab-exercise-1/SarutaFeature.java:1]: good.

**Review by jaxmmmm — COMMENTED** [review]
Added review

File: lab-exercise-1/SarutaFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+      public void feature() {
+          // this is a new feature
+      }
```
Comment (6588041chalisakengkaewpennapa) [inline lab-exercise-1/SarutaFeature.java:4]: this code is empty please improve.

**Pull request closed without merging**